pkg/config/config.go

```diff
@@ -68,6 +68,8 @@ type Config struct {
 	EnableTelemetry          bool          `yaml:"enableTelemetry"`
 	EnableRetinaEndpoint     bool          `yaml:"enableRetinaEndpoint"`
 	EnablePodLevel           bool          `yaml:"enablePodLevel"`
+	EnableStandalone         bool          `yaml:"enableStandalone"`
+	EnableCrictl             bool          `yaml:"enableCrictl"`
```

Comment:
I don't like this config setup like this. Would rather like to see something like
`--enrichment-mode=crictl/azure-vnet-statefile` (please decide on a better flag name).